test/test.baseexternalclient.ts

```diff
@@ -356,14 +357,34 @@ describe('BaseExternalAccountClient', () => {
         const client = new TestExternalAccountClient(
           externalAccountOptionsWithCreds
         );
+        // Listen to tokens events. On every event, push to list of
+        // emittedEvents.
+        client.on('tokens', tokens => {
```

Comment:
BTW, should we add a similar test for downscoped client to assert emitted event?